libs/crypto/tests/unit/mcl_dkg_tests.cpp

```diff
@@ -205,21 +205,14 @@ TEST(MclDkgTests, Signing)
 {
   mcl::bn::initPairing();
 
-  uint32_t committee_size = 5;
-  uint32_t threshold      = 3;
-  bn::G2   group_g;
-  SetGenerator(group_g);
+  uint32_t committee_size = 200;
+  uint32_t threshold      = 101;
 
-  // Construct polynomial of degree threshold - 1
-  std::vector<bn::Fr> vec_a;
-  Init(vec_a, threshold);
-  for (uint32_t ii = 0; ii < threshold; ++ii)
-  {
-    vec_a[ii].setRand();
-  }
+  // outputs[i] is assigned to node with index i in the committee
```

Comment:
I thought the committee indexing only really mattered for the RBC?

Reply:
No, the DKG associates each committee member with an index according to their position in the set of muddle address. This is important for the lagrange interpolation required in computing the group signature, as well as other parts of the DKG.